src/library/dao/playlistdao.h

```diff
@@ -40,6 +40,12 @@ class PlaylistDAO : public QObject, public virtual DAO {
         PLHT_UNKNOWN = -1
     };
 
+    enum class AutoDJSendLoc {
+        TOP,
+        BOTTOM,
+        REPLACE,
+    };
+
```

Comment:
Hm, maybe it should eject the loaded, stopped track, remove it from the top of the queue, and load the new first track in the queue.